diff --git a/node_modules/@moltin/sdk/dist/moltin.esm.js b/node_modules/@moltin/sdk/dist/moltin.esm.js
index 972dd1f..f7d1a5d 100644
--- a/node_modules/@moltin/sdk/dist/moltin.esm.js
+++ b/node_modules/@moltin/sdk/dist/moltin.esm.js
@@ -1,6 +1,4 @@
 import { pluralize, underscore, singularize } from 'inflected';
-import 'fetch-everywhere';
-import 'es6-promise';
 
 function _typeof(obj) {
   if (typeof Symbol === "function" && typeof Symbol.iterator === "symbol") {
