diff --git a/node_modules/apollo-link-http-common/lib/bundle.cjs.js b/node_modules/apollo-link-http-common/lib/bundle.cjs.js
index 60b8d29..56fde73 100644
--- a/node_modules/apollo-link-http-common/lib/bundle.cjs.js
+++ b/node_modules/apollo-link-http-common/lib/bundle.cjs.js
@@ -43,6 +43,12 @@ var parseAndCheckHttpResponse = function (operations) { return function (respons
             parseError.response = response;
             parseError.statusCode = response.status;
             parseError.bodyText = bodyText;
+            console.log(' **** extra logs ***')
+            console.error(operations);
+            console.dir(operaions);
+            console.log(bodyText);
+            console.log(' **** end extra logs ***')
+
             return Promise.reject(parseError);
         }
     })
diff --git a/node_modules/apollo-link-http-common/lib/index.js b/node_modules/apollo-link-http-common/lib/index.js
index 05cd095..ba3eede 100644
--- a/node_modules/apollo-link-http-common/lib/index.js
+++ b/node_modules/apollo-link-http-common/lib/index.js
@@ -40,6 +40,12 @@ exports.parseAndCheckHttpResponse = function (operations) { return function (res
             parseError.response = response;
             parseError.statusCode = response.status;
             parseError.bodyText = bodyText;
+            console.log(' **** extra logs ***')
+            console.error(operations);
+            console.dir(operaions);
+            console.log(bodyText);
+            console.log(' **** end extra logs ***')
+
             return Promise.reject(parseError);
         }
     })
